smps.proto

```diff
@@ -2,6 +2,9 @@ syntax = "proto3";
 package SMPS;
 
 import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "spec;spec";
```

Comment:
Should this be something like `layer5.smps.spec`?

Reply:
We could do that, shall I ?

Reply:
I suppose so. I think a better suggestion is `layer5io.service-mesh-performance-specification`

Reply:
Should we switch to SMPS here too?

Reply:
I would like that. I hesitate to suggest it in that my understanding is that this becomes a filesystem reference in `$GOPATH/SRC` when compiling the proto file (using `protoc`), but I'm not sure of all the considerations here. If `smps` works just as well, that's slick.

Reply:
I didn't get you exactly here.. but when I compile this proto file I get the output in here itself, In the spec folder.

Reply:
I didn’t want to let the night slip by without helping get this merged. 

The consideration that I’m doing a poor job of articulating might be prevalent only when third parties are importing this package. I’m not sure.